mlptrain/utils.py

```diff
@@ -298,3 +300,36 @@ def convert_ase_energy(
         raise ValueError(f'Unknown energy units: {units}')
 
     return energy_array
+
+
+def npz_to_xyz(npz_filename: str) -> None:
+    """
+    Converts a Trajectory or ConfigurationSet saved as an .npz file to a .xyz file.
+
+    ---------------------------------------------------------------------------
+    Arguments:
+
+        npz_file: (str) Name of the .npz file to be converted, eg. "my_data.npz"
+
+
+    Creates a .xyz file named the same way, eg. -> 'my_data.xyz'
+
+    Returns: None
+    """
+
+    if not npz_filename.endswith('.npz'):
+        raise ValueError('Input filename must end with .npz extension.')
+
+    xyz_filename = npz_filename.replace('.npz', '.xyz')
```

Comment:
Better than replace would be to use a regular expression:
```suggestion
    import re
    xyz_filename = re.sub('.npz$', '.xyz',npz_filename)
```

This will ensure that only the last .npz will be changed into .xyz. This is just a more general solution.

More information is here: https://realpython.com/replace-string-python/#leverage-resub-to-make-complex-rules